### ISSUE TITLE
I'm having trouble in inference

### ISSUE THREAD

**Issue by newlc-lc** [MJITG/PyTorch-HITNet-Hierarchical-Iterative-Tile-Refinement-Network-for-Real-time-Stereo-Matching#10]
After Kitti trained the network, the model was much less effective in inference. But it's not an overfit. Because I'm using the test set to inference.and The dataload for test and inference is the same. Does anyone have a problem with that.
